### PR TITLE
DOC: Extend Doxygen main page toolkit description

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: Documentation/Doxygen/MainPage.dox [hunk 1]
```diff
@@ -6,10 +6,21 @@
...
+ *
+ * Segmentation is the process of identifying and classifying data found in a
+ * digitally sampled representation. Typically the sampled representation is an
+ * image acquired from such medical instrumentation as CT or MRI scanners.
+ * Registration is the task of aligning or developing correspondences between
+ * data. For example, in the medical environment, a CT scan may be aligned with a
+ * MRI scan in order to combine the information contained in both.
+ *
+ * ITK was originally created to analyze 3D images from the Visible Human
+ * Project. ITK is now widely applied across medically imaging, microscopy,
```
Comment (dzenanz) [inline Documentation/Doxygen/MainPage.dox]: medically -> medical

File: Documentation/Doxygen/MainPage.dox [hunk 2]
```diff
@@ -6,10 +6,21 @@
...
+ * and remote sensing domains. ITK is actively developed by a community of
+ * research software engineers from around world.
```
Comment (dzenanz) [inline Documentation/Doxygen/MainPage.dox]: around world -> around **the** world

File: Documentation/Doxygen/MainPage.dox (hunk 1 above)
Comment (thewtex) [reply Documentation/Doxygen/MainPage.dox]: Fixed!

File: Documentation/Doxygen/MainPage.dox (hunk 2 above)
Comment (thewtex) [reply Documentation/Doxygen/MainPage.dox]: Good catch- fixed!

**Pull request merged**